torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -238,6 +281,17 @@ Operation BroadOp(const Node* node) {
   };
 }
 
+const RegisterOperators MKLDNNHardSwishOpReg({
+    torch::jit::Operator(
+        "aten::MKLDNNHardSwish(Tensor a) -> Tensor",
```

Comment:
We should have a way of inplacing this : https://github.com/pytorch/pytorch/blob/ce32d3d54b4bad74ecc4b44f68b1ca94b734e7ef/torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp#L180.

Maybe just by adding an `inplace=true` attribute to the node